libr/core/cmd_anal.c

```diff
@@ -39,9 +39,81 @@ static bool anal_is_bad_call(RCore *core, ut64 from, ut64 to, ut64 addr, ut8 *bu
 }
 #endif
 
+static void cmd_esil_mem(RCore *core, const char *input) {
+	ut64 curoff = core->offset;
+	ut64 addr = 0x100000;
+	ut32 size = 0xf0000;
+	char name[128];
+	RCoreFile *cf;
+	RFlagItem *fi;
+	const char *sp;
+	char uri[32];
+	char nomalloc[256];
+	char *p;
+	if (*input == '?') {
+		eprintf ("Usage: [addr] [size] [name]\n");
+		eprintf ("Default: 0x100000 0xf0000\n");
+		return;
+	}
+
+	p = strncpy (nomalloc, input, 255);
+	if ((p = strchr (p, ' '))) {
+		while (*p == ' ') p++;
+		addr = r_num_math (core->num, p);
+		if ((p = strchr (p, ' '))) {
+			while (*p == ' ') p++;
+			size = (ut32)r_num_math (core->num, p);
+			if (size < 1)
```

Comment:
missing braces. also if size is ut32 the min value will be 0. what about making that signed to catch >2GB as invalid?